lua/telescope/actions/init.lua

```diff
@@ -343,28 +343,25 @@ end
 --- Close the Telescope window, usually used within an action
 ---@param prompt_bufnr number: The prompt bufnr
 actions.close = function(prompt_bufnr)
-  actions._close(prompt_bufnr, false)
-end
-
---- Close the Telescope window and specify if you want to keep insert mode or not
----@param prompt_bufnr number: The prompt bufnr
----@param keepinsert boolean: Remain in INSERT mode if true
-actions._close = function(prompt_bufnr, keepinsert)
   action_state.get_current_history():reset()
   local picker = action_state.get_current_picker(prompt_bufnr)
   local prompt_win = state.get_status(prompt_bufnr).prompt_win
```

Comment:
shouldn't we validate this value? or bail out early otherwise?

Reply:
?

Reply:
```lua
if not vim.api.nvim_win_is_valid(prompt_win) then
  -- nothing to do | raise error
  return
end
```

Reply:
worked for the past year. If you call this function outside of a telescope action, its undefined behavior